darts/models/forecasting/torch_forecasting_model.py

```diff
@@ -170,13 +170,13 @@ def __init__(
 
             .. deprecated:: v0.17.0
                 ``torch_device_str`` has been deprecated in v0.17.0 and will be removed in a future version.
-                Instead, specify this with keys ``"accelerator", "gpus", "auto_select_gpus"`` in your
+                Instead, specify this with keys ``"accelerator", "devices", "auto_select_gpus"`` in your
                 ``pl_trainer_kwargs`` dict. Some examples for setting the devices inside the ``pl_trainer_kwargs``
                 dict:
 
                 - ``{"accelerator": "cpu"}`` for CPU,
-                - ``{"accelerator": "gpu", "gpus": [i]}`` to use only GPU ``i`` (``i`` must be an integer),
-                - ``{"accelerator": "gpu", "gpus": -1, "auto_select_gpus": True}`` to use all available GPUS.
+                - ``{"accelerator": "gpu", "devices": [i]}`` to use only GPU ``i`` (``i`` must be an integer),
+                - ``{"accelerator": "gpu", "devices": -1, "auto_select_gpus": True}`` to use all available GPUS.
```

Comment:
could you adapt the code parts as well were we add the "gpus"?
- https://github.com/unit8co/darts/blob/06824b29c094250b55bd121cd44d42dac87f08af/darts/models/forecasting/torch_forecasting_model.py#L326

We probably should check the pytorch lightning version to either add "gpus" or "device". You can do to this similarly to https://github.com/unit8co/darts/blob/06824b29c094250b55bd121cd44d42dac87f08af/darts/models/forecasting/pl_forecasting_module.py#L397

Reply:
updated